crates/anstyle-roff/Cargo.toml

```diff
@@ -26,3 +26,4 @@ cansi = "2.2.1"
 
 [dev-dependencies]
 snapbox = "0.3.3"
+snapbox-macros = "0.3.0"
```

Comment:
Why was this needed?

Reply:
The CI was failing because of snapbox-macros version requiring a higher MSRV than the CI was allowing.  Hoping that pinning it would fix the issue.